src/backend/defs/rest-api/objects/Discussion.json

```diff
@@ -5,10 +5,10 @@
       "type": "integer",
       "format": "int32"
     },
-    "contacts": {
+    "participants": {
       "type": "array",
       "items": {
-        "$ref": "Contact.json"
+        "$ref": "Participant.json"
```

Comment:
this will break current GET /discussions route
- adapt parameters to this new one
- postpone this update when we will rework this route and related